manifests/init.pp

```diff
@@ -174,8 +175,11 @@
     }
   }
 
+  validate_bool($service_start_with_system)
+
   service {'pgbouncer':
     ensure    => running,
```

Comment:
actually, you should conditional on ensure. If enable is true, enable is running else its stopped